edq/cli/config/set.py

```diff
@@ -0,0 +1,81 @@
+"""
+Set a configuration option.
+"""
+
+import argparse
+import sys
+import typing
+
+import edq.core.argparser
+import edq.core.config
+
+def run_cli(args: argparse.Namespace) -> int:
+    """ Run the CLI. """
+
+    config: typing.Dict[str, str] = {}
+    for config_option in args.config_to_set:
+        (key, value) = edq.core.config._parse_cli_config_option(config_option)
```

Comment:
Why the private name (underscore prefix)?
Usually we do this to indicate a function that takes extra specialized knowledge to use.
But this seems like a function anyone can use.